src/lib/mina_lib/config.ml

```diff
@@ -60,6 +60,9 @@ type t =
   ; uptime_submitter_keypair : Keypair.t option [@default None]
   ; uptime_send_node_commit : bool [@default false]
   ; stop_time : int
+  ; file_log_level : Logger.Level.t [@default Logger.Level.Info]
```

Comment:
I suggest have documentations for these new options

Reply:
It's documented in src/lib/cli_lib/flag.ml as this is common cli flag. Do you think it is enough? or should i basically repeat here, what's there?

Reply:
Ok for now if documented somewhere else.